docusaurus.config.js

```diff
@@ -59,7 +74,7 @@ module.exports = {
     announcementBar: {
       id: "camunda8",
       content:
-        '📣 <b><a target="_blank" rel="noopener noreferrer" href="https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral&utm_content=banner">Sign-Up</a></b> for a free account to start orchestrating business processes today.',
+        '📣 <b><a target="_blank" rel="noopener noreferrer" href="https://signup.camunda.com/accounts?utm_source=1_3.docs.camunda.io&utm_medium=referral&utm_content=banner">Sign-Up</a></b> for a free account to start orchestrating business processes today.',
```

Comment:
🤷 I updated these referral URLs because i noticed them. Not sure if we care.

Reply:
I would say let's remove the banner, or use it as a clear visual aid that you are in old, old, old docs.

Reply:
I like the idea to use it as a clear reminder that you are in old docs. I'll play with this.

Reply:
I gave you a code snippet in the other [PR](https://github.com/camunda/camunda-platform-docs/pull/2128#issuecomment-1557783802). 🙈  You can have it here too

```
announcementBar: {
      id: "unsupported",
      content:
        '🚨 These docs are unsupported and unmaintained. Looking for <b><a target="_blank" rel="noopener noreferrer" href="https://docs.camunda.io">supported versions?</a></b> 🚨',
      backgroundColor: "#FFC600",
      textColor: "#434343",
      isCloseable: false,
    },
```